src/components/contact/assets/socialMedia.scss

```diff
@@ -0,0 +1,16 @@
+@import '~theme/base/variables';
+
+.socialMedia {
+  display: flex;
+  margin-top: 20px;
+  flex-flow: row-reverse;
+  .Icon__Wrapper {
+    margin: 10px;
+    padding: 15px;
+    border: 2px solid $color-light-green;
```

Comment:
Could you sort the css from A-Z? 🙄

Reply:
@crysfel It's difficult. I will give this the least priority.